detectors/node/opentelemetry-resource-detector-docker/src/detectors/DockerDetector.ts

```diff
@@ -25,10 +25,12 @@ import * as fs from 'fs';
 import * as util from 'util';
 import { diag } from '@opentelemetry/api';
 
-export class DockerCGroupV1Detector implements Detector {
+export class DockerDetector implements Detector {
   readonly CONTAINER_ID_LENGTH = 64;
   readonly DEFAULT_CGROUP_PATH = '/proc/self/cgroup';
```

Comment:
Should we call it `DEFAULT_CGROUP_V1_PATH` to make it consistent with the new definition?

Reply:
Makes sense